recipes/neo4j-python-driver/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name='neo4j-python-driver' %}
+{% set version='1.1.0rc1' %}
```

Comment:
Latest version is 1.3.1. Is there a reason to package an older release candidate?